src/pages/studyDiary/StudyDiaryEditPage.jsx

```
@@ -71,17 +77,74 @@ function StudyDiaryEditPage() {
+      const line = cm.getLine(cursor.line);
+      
+      // 현재 줄이 비어있지 않으면 새 줄에서 시작
+      const isEmptyLine = !line || line.trim() === '';
+      const insertText = isEmptyLine ? markdownLink : `\n${markdownLink}`;
+      
+      // 현재 위치에 이미지 삽입
+      cm.replaceRange(insertText, cursor);
+      
+      // 이미지 다음 줄로 커서 이동
+      const newCursor = { 
+        line: cursor.line + (isEmptyLine ? 0 : 1), 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)